bgpd/bgp_route.c

```diff
@@ -11059,18 +11167,34 @@ static int bgp_table_stats_walker(struct thread *t)
 }
 
 static int bgp_table_stats(struct vty *vty, struct bgp *bgp, afi_t afi,
-			   safi_t safi)
+			   safi_t safi, struct json_object *json_array)
 {
 	struct bgp_table_stats ts;
 	unsigned int i;
+	int ret = CMD_SUCCESS;
+	char temp_buf[20];
```

Comment:
just curious: why 20?

Reply:
well, the maximume used seems to be 12.2f aka 15 bytes.
so 20 byte seems to me correct.
If you have some advice to give me, please let me know.